backend/src/main/resources/application.properties

```diff
@@ -15,3 +15,15 @@ application.access-token-validity-seconds=18000
 application.jwt-signing-key=211822d8-462a-4cc5-b2f3-2534789b9062
 
 application.max-candidate-pending-application=3
+
+spring.mail.host = smtp.gmail.com
+spring.mail.port = 465
+spring.mail.username = diningtogether123
+spring.mail.password = cxwkxuitfiodaxvl
```

Comment:
It is a bad idea to expose your gmail password to public. You should read it from environment variable or use multiple properties files and exclude the file with passwords from git.

I suggest you to change your password now.